src/Equinor.ProCoSys.IPO.Command/InvitationCommands/EditInvitation/EditInvitationCommandValidator.cs

```diff
@@ -52,11 +52,15 @@ public EditInvitationCommandValidator(IInvitationValidator invitationValidator,
 
             RuleForEach(command => command.UpdatedParticipants)
                 .MustAsync((command, participant, _, cancellationToken) => ParticipantToBeUpdatedMustExist(participant, command.InvitationId, cancellationToken))
-                .WithMessage((command, participant) =>
-                    $"Participant with ID does not exist on invitation! Participant={participant}")
+                .WithMessage(_ => $"Participant with ID does not exist on invitation!")
+                .Must(participant => participant.SortKey >= 0)
```

Comment:
Maybe stupid question, but isn't the test in the code for SortKey > 1?

Reply:
The check in Handler for SortKey > 1 must be something else. 

I've just moved the rule from src/Equinor.ProCoSys.IPO.Command/InvitationCommands/ParticipantsForCommandValidator.cs which is now deleted, becauase the rule was never validated. The writer of code has misunderstood when Validators in FluentValidation kick in.